dispatch.yaml

```diff
@@ -11,7 +11,8 @@ dispatch:
 - url: "*/tasks/email-subscribers"
   service: notifier
 
-- url: "*/hello"
-  service: app-py3
+- url: "*/tasks/outbound-email"
```

Comment:
Wouldn't it better to make app-py3 the default service? I don't see too much of a difference [in the documentation]9https://cloud.google.com/appengine/docs/standard/nodejs/configuration-files#the_default_service) but it makes more sense

Reply:
I would like it to be the default service, but requests to /_ah/* always go to the default service, and we need them to be in py2 to use the GAE mail API.  There's an existing comment about that in dispatch.yaml.

Reply:
Yea I just remembered that